packages/lesswrong/lib/collections/advisorRequests/collection.ts

```diff
@@ -0,0 +1,18 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { userOwns, userCanDo } from '../../vulcan-users/permissions';
+import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
+import { getDefaultMutations, MutationOptions } from '../../vulcan-core/default_mutations';
+
+export const AdvisorRequests: AdvisorRequestsCollection = createCollection({
```

Comment:
Although the primary form isn't on the forum anymore, we're still using this collection to track who has requested a chat (if they have a forum account).